### PR TITLE
[graphql/rpc] impl simple client

### PR DESCRIPTION
**Pull request by oxade**
## Description 

Adds simple client to be used in tests and examples in coming PRs

## Test Plan 

Unit test

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Review by stefan-mysten — APPROVED** [review]

File: crates/sui-graphql-rpc/src/client/simple_client.rs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    tokio::time::sleep(std::time::Duration::from_secs(10)).await;
+    let client = SimpleClient::new("http://127.0.0.1:8000/");
+    let query = r#"
+        query {
+            chainIdentifier
+        }
+    "#;
+    let res = client.execute(query.to_string()).await.unwrap();
+    let exp =
+        r#"{"data":{"chainIdentifier":"4c78adac"},"extensions":{"usage":{"nodes":1,"depth":1}}}"#;
```
Comment (stefan-mysten) [inline crates/sui-graphql-rpc/src/client/simple_client.rs:53]: Will this chainIdentifier change when we update our read-only snapshot?

Comment (oxade) [reply crates/sui-graphql-rpc/src/client/simple_client.rs:53]: no, `chainIdentifier` in tied to the underlying network.

Comment (stefan-mysten) [reply crates/sui-graphql-rpc/src/client/simple_client.rs:53]: Lovely, thanks.

**Comment by stefan-mysten** [comment]
Looks great, thanks! I will add some more examples tomorrow.

**Comment by bmwill** [comment]
@oxade It appears this is regularly breaking CI since its landed

**Pull request merged**